dev/io.openliberty.microprofile.telemetry.1.0.internal/bnd.bnd

```diff
@@ -52,14 +52,17 @@ Service-Component: \
 # created for other API classes
 Import-Package: \
 	io.opentelemetry.api.common,\
+  com.ibm.websphere.ras,\
+  com.ibm.websphere.ras.annotation,\
```

Comment:
Were these necessary? I _thought_ they should be captured automatically because you import `*` and `com.ibm.ws.logging.core` is on the build path, but I may be missing something there.

Reply:
No you are right. I was going with what I did in the other bundle but it was necessary there